### ISSUE TITLE
Use Redis for sessions

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#812]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Switch from `connect-azuretables` to `connect-redis`.

### Description
We should look into an alternative to [connect-azuretables](https://www.npmjs.com/package/connect-azuretables). Last version was published 2 years ago and got 107 weekly downloads (most of them are probably from the Did team). [connect-redis](https://www.npmjs.com/package/connect-redis) looks like a good alternative.

https://stackoverflow.com/questions/8749907/what-is-a-good-session-store-for-a-single-host-node-js-production-app

_Originally posted by @olemp in https://github.com/Puzzlepart/did/issues/792#issuecomment-775121507_
